src/java.base/share/classes/sun/security/ec/SunEC.java

```diff
@@ -403,68 +467,135 @@ void putEntries() {
+                ATTRS));
+            putService(new ProviderServiceA(this, "Signature",
+                "SHA3-384withECDSA", "sun.security.ec.ECDSASignature$SHA3_384",
+                ATTRS));
+            putService(new ProviderServiceA(this, "Signature",
+                "SHA3-512withECDSA", "sun.security.ec.ECDSASignature$SHA3_512",
+                ATTRS));
 
-        putService(new ProviderService(this, "Signature",
-             "SHA3-224withECDSAinP1363Format",
-             "sun.security.ec.ECDSASignature$SHA3_224inP1363Format"));
-        putService(new ProviderService(this, "Signature",
-             "SHA3-256withECDSAinP1363Format",
-             "sun.security.ec.ECDSASignature$SHA3_256inP1363Format"));
-        putService(new ProviderService(this, "Signature",
-            "SHA3-384withECDSAinP1363Format",
-            "sun.security.ec.ECDSASignature$SHA3_384inP1363Format"));
-        putService(new ProviderService(this, "Signature",
-            "SHA3-512withECDSAinP1363Format",
-            "sun.security.ec.ECDSASignature$SHA3_512inP1363Format"));
+            putService(new ProviderService(this, "Signature",
+                "NONEwithECDSAinP1363Format",
+                "sun.security.ec.ECDSASignature$RawinP1363Format"));
+            putService(new ProviderService(this, "Signature",
+                "SHA1withECDSAinP1363Format",
+                "sun.security.ec.ECDSASignature$SHA1inP1363Format"));
+            putService(new ProviderService(this, "Signature",
+                "SHA224withECDSAinP1363Format",
+                "sun.security.ec.ECDSASignature$SHA224inP1363Format"));
+            putService(new ProviderService(this, "Signature",
+                "SHA256withECDSAinP1363Format",
+                "sun.security.ec.ECDSASignature$SHA256inP1363Format"));
+            putService(new ProviderService(this, "Signature",
+                "SHA384withECDSAinP1363Format",
+                "sun.security.ec.ECDSASignature$SHA384inP1363Format"));
+            putService(new ProviderService(this, "Signature",
+                "SHA512withECDSAinP1363Format",
+                "sun.security.ec.ECDSASignature$SHA512inP1363Format"));
+
+            putService(new ProviderService(this, "Signature",
+                "SHA3-224withECDSAinP1363Format",
+                "sun.security.ec.ECDSASignature$SHA3_224inP1363Format"));
+            putService(new ProviderService(this, "Signature",
+                "SHA3-256withECDSAinP1363Format",
+                "sun.security.ec.ECDSASignature$SHA3_256inP1363Format"));
+            putService(new ProviderService(this, "Signature",
+                "SHA3-384withECDSAinP1363Format",
+                "sun.security.ec.ECDSASignature$SHA3_384inP1363Format"));
+            putService(new ProviderService(this, "Signature",
+                "SHA3-512withECDSAinP1363Format",
+                "sun.security.ec.ECDSASignature$SHA3_512inP1363Format"));
+        }
```

Comment:
On line 609, I expect "Natice" should be "Native" which might explain the test failures in https://openj9-jenkins.osuosl.org/job/Grinder/3559:
```
14:30:31  java.lang.RuntimeException: No exception was thrown. Expected: java.lang.IllegalStateException
14:30:31  	at ECDHKeyAgreementParamValidation.testInitWithInvalidKey(ECDHKeyAgreementParamValidation.java:69)
14:30:31  	at ECDHKeyAgreementParamValidation.main(ECDHKeyAgreementParamValidation.java:103)

14:30:31  java.lang.RuntimeException: No exception was thrown. Expected: java.security.InvalidKeyException
14:30:31  	at ECDHKeyAgreementParamValidation.testDoPhaseWithInvalidKey(ECDHKeyAgreementParamValidation.java:91)
14:30:31  	at ECDHKeyAgreementParamValidation.main(ECDHKeyAgreementParamValidation.java:110)
```
I see the same typo in jdk 11, 17, 21. and 22.

Reply:
I created pull requests to fix jdk 11, 17, and 21:
* https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/774
* https://github.com/ibmruntimes/openj9-openjdk-jdk17/pull/346
* https://github.com/ibmruntimes/openj9-openjdk-jdk21/pull/151

Reply:
Fixed that. Wonder how we missed that. The test, however, seems to have been added recently and has to do with `ECDHKeyAgreement`. I'll open a separate issue to fix that.

Reply:
The back port will address those.

Reply:
> I see the same typo in jdk 11, 17, 21. and 22.
> I created pull requests to fix jdk 11, 17, and 21:

Is the `22` back port coming?

Reply:
The fix for jdk22 is in https://github.com/ibmruntimes/openj9-openjdk-jdk22/pull/54.

Reply:
The test failed again, so I created https://github.com/eclipse-openj9/openj9/issues/19437.